packages/ckeditor5-ui/tests/manual/dialog/dialog-body-scroll-lock.html

```diff
@@ -0,0 +1,53 @@
+<form>
+	<fieldset>
+		<legend>Is document scrollable?</legend>
+		<div>
+			<input type="radio" id="yes" name="isScrollable" value="yes" />
+			<label for="yes">yes</label>
+		</div>
+		<div>
+			<input type="radio" id="no" name="isScrollable" value="no" checked />
+			<label for="no">no</label>
+		</div>
+	</fieldset>
+	<fieldset>
+		<legend>Has &lt;body&gt; position?</legend>
+		<div>
+			<input type="radio" id="yes" name="hasPosition" value="yes" />
+			<label for="yes">yes</label>
+		</div>
+		<div>
+			<input type="radio" id="no" name="hasPosition" value="no" checked />
+			<label for="no">no</label>
```

Comment:
Ids are duplicated so labels don't work well.

Reply:
Addressed in d03717ac.